activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/rest/controller/ProjectController.java

```diff
@@ -123,6 +123,15 @@ public void exportProject(
                             attachment);
     }
 
+    @Override
+    public void validateProject(
+            HttpServletResponse response,
```

Comment:
`response` parameter doesn't seem to be used.

Reply:
Updated, thanks